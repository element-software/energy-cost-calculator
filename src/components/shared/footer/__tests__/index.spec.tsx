import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Footer from "..";

describe("Footer", () => {
  it("should render the component successfully", () => {
    const { baseElement } = render(<Footer />);
    expect(baseElement).toMatchSnapshot();
  });
});
