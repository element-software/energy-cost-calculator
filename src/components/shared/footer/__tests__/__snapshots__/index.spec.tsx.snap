// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > should render the component successfully 1`] = `
<body>
  <div>
    <footer
      class="mt-10 text-center text-sm text-foreground/70 gap-3 flex flex-row justify-center"
    >
      <p>
        Iqbal Ibrahim Â© 
        2025
      </p>
      <a
        href="https://linkedin.com/in/iqbalibrahim"
        rel="noopener noreferrer"
        target="_blank"
      >
        LinkedIn
      </a>
      <a
        href="https://github.com/elementsoftware"
        rel="noopener noreferrer"
        target="_blank"
      >
        GitHub
      </a>
      <a
        href="https://iqbalibrahim.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        Personal
      </a>
      <a
        href="https://element-software.co.uk"
        rel="noopener noreferrer"
        target="_blank"
      >
        Business
      </a>
    </footer>
  </div>
</body>
`;
